rust/ql/src/queries/security/CWE-089/SqlInjection.ql

```diff
@@ -0,0 +1,35 @@
+/**
+ * @name Database query built from user-controlled sources
+ * @description Building a database query from user-controlled sources is vulnerable to insertion of malicious code by the user.
```

Comment:
Perhaps, rather than using `the user`.
```suggestion
 * @description Building a database query from user-controlled sources is vulnerable to insertion of malicious code by attackers.
```